chapters/en/module8.md

```diff
@@ -142,11 +146,11 @@ $$\Sigma F = m \frac{dV}{dt}$$
 </opt>
 
 <opt text="<code>:label:</code> should be changed to <code>label:</code>">
-<code>:label:</code> is actually the right way to give a label to an equation.
+<code>:label:</code> is actually the right way to give a label to an equation. 
 </opt>
 
 <opt text="<code>:label:</code> should be replaced by <code>:name:</code>">
-Using <code>:name:</code> is incorrect. You've probably mistaken this with <code>name:</code> in a <code>```{figure} ```</code> block.
+Using <code>:name:</code> is incorrect. You've probably mistaken this with <code>name:</code> in a <code>```{figure} ```</code> block. 
 </opt>
```

Comment:
Thanks for the note. I tried both alternative scenarios; i.e. using `:name:` for figures (instead of `name:`) and `label:` for equations (instead of `:label:`), and the rendering broke in both cases. There's also a distinction between the `{image}` and `{figure}` directives, which is why both single and double colon notations are seen in the documentation.

Reply:
Ah ok great, thanks for explaining that difference!